src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```
@@ -1290,7 +1291,9 @@ because xbuild doesn't support framework reference assemblies.
 	DependsOnTargets="$(CoreResolveReferencesDependsOn);_CreatePropertiesCache;_CheckForDeletedResourceFile;_ComputeAndroidResourcePaths;_UpdateAndroidResgen;_CreateManagedLibraryResourceArchive;_GenerateJavaDesignerForComponent"> 
 </Target>
 
-<Target Name="UpdateAndroidResources" DependsOnTargets="_SetupDesignTimeBuildForCompile;_ManagedUpdateAndroidResgen;_UpdateAndroidResources" />
+<Target Name="UpdateAndroidResources" 
+	Condition=" '$(DesignTimeBuild)' == 'false' or '$(BackgroundBuildSupported)' != 'true' or '$(BackgroundBuild)' == 'true' "
```

Minor change:
```suggestion
	Condition=" '$(DesignTimeBuild)' != 'True' Or '$(BackgroundBuildSupported)' != 'True' Or '$(BackgroundBuild)' == 'True' "
```
We're not consistent everywhere yet, but we've started capitalizing `True`, `False`, `Or`, and `And`.

I also think we should check `'$(DesignTimeBuild)' != 'True'` in case it were blank.

---

But I thought it can never be blank, since this target depends on the `_SetupDesignTimeBuildForCompile`? Agreed on consistency.